packages/git/src/index.ts

```diff
@@ -296,5 +296,6 @@ export async function getCurrentCommitId({
     await spawn("git", ["show", `--format="%h"`, "--no-patch"], { cwd })
```

Comment:
I wonder if the reason we end up with those quotation marks isn't that we include them here in the `format`. I've checked on my machine and this works fine too:
```suggestion
    await spawn("git", ["show", "--format=%h", "--no-patch"], { cwd })
```

Alternatively we could also just switch to using this:
```suggestion
    await spawn("git", ["rev-parse", "--short", "HEAD"], { cwd })
```
It seems to be better suited for this job anyway.

Reply:
Both suggestion seem to solve the problem. I agree that the second one seems to be better for this. I updated the PR.